pyproject.toml

```diff
@@ -54,9 +54,9 @@ targets.wheel.packages = ["src/wiscopy"]
 [tool.pytest.ini_options]
 testpaths = ["tests"]
 
-[tool.pixi.project]
+[tool.pixi.workspace]
```

Comment:
This is just a remapping to the new standard. c.f. https://github.com/prefix-dev/pixi/releases/tag/v0.43.0